src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/IndexUtils.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.util
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.IndexStateManagementIndices
+import org.elasticsearch.action.ActionListener
+import org.elasticsearch.action.admin.indices.mapping.put.PutMappingRequest
+import org.elasticsearch.action.support.master.AcknowledgedResponse
+import org.elasticsearch.client.IndicesAdminClient
+import org.elasticsearch.cluster.ClusterState
+import org.elasticsearch.cluster.metadata.IndexMetaData
+import org.elasticsearch.common.xcontent.LoggingDeprecationHandler
+import org.elasticsearch.common.xcontent.NamedXContentRegistry
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentType
+
+class IndexUtils {
+    companion object {
+        @Suppress("ObjectPropertyNaming")
+        const val _META = "_meta"
+        const val SCHEMA_VERSION = "schema_version"
+        const val NO_SCHEMA_VERSION = 1L
```

Comment:
nit: Should we change this to DEFAULT_SCHEMA_VESRION? That seems to describe the value better, i.e. if there is no schema version specified we use the default schema version.

Reply:
changed.